components/text.js

```diff
@@ -294,8 +307,8 @@ export default memo(function Text ({ rel, imgproxyUrls, children, tab, itemId, o
           },
           img: Img
         }}
-        remarkPlugins={[gfm, mention, sub]}
-        rehypePlugins={[rehypeInlineCodeProperty]}
+        remarkPlugins={[gfm, mention, sub, rehype]}
+        rehypePlugins={[rehypeInlineCodeProperty, rehypeRaw, [rehypeSanitize, rehypeSanitizeSchema]]}
```

Comment:
This order breaks inline code. Even single backticks are now parsed a blocks. This fixes it:

```suggestion
        rehypePlugins={[rehypeRaw, [rehypeSanitize, rehypeSanitizeSchema], rehypeInlineCodeProperty]}
```

Reply:
Interesting behavior from just changing the order of items.

Reply:
It seems to be a pipeline so changing the order means that `rehypeInlineCodeProperty` will be executed last. If it's executed first, `rehypeRaw` overwrites the inline property. At least that's how I explained this to myself.